### DISCUSSION TITLE
Foundry V11, is it safe to upgrade?

### DISCUSSION THREAD

**Discussion by avorial** [3darkman/foundryvtt-chroniclesystem#60] [Q&A]
It's that time of year again, during the V10 upgrade you noted that everything should be stable until V12 is that still the case?

https://www.reddit.com/r/FoundryVTT/comments/13qv62l/foundryvtt_v11_stable_build_299_release/

**Comment by 3darkman** [comment]
I think it's a little late, but yes, it was safe :/
